js/router.js

```diff
@@ -48,6 +48,10 @@ module.exports = Backbone.Router.extend({
         this.navigate(translatedRoute, { trigger: true });
       });
     });
+    
+    history.size = -1;
```

Comment:
I think it's a REALLY bad idea to muck with a core JS object like History. Your are adding properties that don't exist on the object now, but the API may change at any time. If in the future the properties are added that you are augmenting, you will be blowing away the props of a core JS object.

It is almost alway a very bad idea to mess with core JS objects.